feature/settings/src/main/java/com/google/samples/apps/nowinandroid/feature/settings/SettingsDialog.kt

```diff
@@ -145,7 +169,22 @@ private fun SettingsPanel(
             onClick = { onChangeThemeBrand(ANDROID) }
         )
     }
-    SettingsDialogSectionTitle(text = "Dark mode preference")
+    if (settings.brand == DEFAULT && supportDynamicColor) {
+        SettingsDialogSectionTitle(text = stringResource(R.string.dynamic_color_preference))
+        Column(Modifier.selectableGroup()) {
+            SettingsDialogThemeChooserRow(
+                text = stringResource(string.dynamic_color_yes),
+                selected = settings.useDynamicColor == true,
```

Comment:
Nit: can this be simplified to:
```suggestion
                selected = settings.useDynamicColor,
```